core/src/main/java/org/apache/sdap/mudrod/main/MudrodEngine.java

```diff
@@ -391,7 +391,7 @@ public static void main(String[] args) {
       me.end();
     } catch (Exception e) {
       HelpFormatter formatter = new HelpFormatter();
-      formatter.printHelp("MudrodEngine: 'dataDir' argument is mandatory. " + "User must also provide an ingest method.", options, true);
+      formatter.printHelp("MudrodEngine: 'dataDir' argument is mandatory. " + "User must also provide an ingest method.", true);
```

Comment:
@echyam this formatter should really just be replaced with a throw statement for the exception.